src/main/java/gtPlusPlus/xmod/gregtech/common/helpers/ChargingHelper.java

```diff
@@ -384,6 +388,18 @@ else if ((mitemCurrentCharge + (mVoltageIncrease*5)) <= (mItemMaxCharge - (mVolt
 
 				}
 			}
+			else if (isItemValidRF(mTemp)) {
+				try {
+					IEnergyContainerItem rfItem = (IEnergyContainerItem)mTemp.getItem();
+					long RF = Math.min(rfItem.getMaxEnergyStored(mTemp) - rfItem.getEnergyStored(mTemp), mEntity.getEUVar() * mEUtoRF / 100L);
+					RF = rfItem.receiveEnergy(mTemp, RF > Integer.MAX_VALUE ? Integer.MAX_VALUE : (int) RF, false);
+					RF = RF * 100L / mEUtoRF;
```

Comment:
Rename variable to something different like `usedEU`. Also I think `*100` is not needed here? Also it's magic number.

Reply:
Afaik the 100 is necessary as the conversion rate is given as 360 instead of 3.6 in Gregtech API. I copied this piece from somewhere else and its the same

Reply:
Ah didn't notice, it's fine